linear_search_algorithm.rb

```diff
@@ -0,0 +1,14 @@
+def linearSearch(array,target)
+
+    i = 0 
+    while i < array.length 
+        if array[i] == target 

```

Comment:
you can also do `array.each_with_index do |value, index|`

Reply:
so linear search is basically just looping through and finding the number if it is there

Reply:
yes of course we can use it, but I have been using .each for most of my problem solving. thought i  could use  while for this. Yes linear search is just an simple iteration search algorithm were we iterate through every element on the  list until we find the match to the target.